sonic-sensormond/tests/mocked_libs/swsscommon/swsscommon.py

```diff
@@ -0,0 +1,56 @@
+'''
```

Comment:
Same here, can we leverage the existing file?

Reply:
I have cleaned up the code a bit and removed the other file.